src/tools/r2rdump/CoreDisTools.cs

```diff
@@ -25,6 +27,10 @@ public enum TargetArch
         [DllImport("coredistools.dll")]
         public static extern void DumpCodeBlock(IntPtr Disasm, ulong Address, IntPtr Bytes, int Size);
 
+        [DllImport("coredistools.dll")]
```

Comment:
I recommend making a static readonly string to hold the name of coredistools.dll. It prevents typos, and makes it a little easier to change in the future.